frontend-web/webclient/app/Project/Resources.tsx

```diff
@@ -178,6 +183,122 @@ const Resources: React.FunctionComponent = () => {
     );
 };
 
+type WalletStore = {
+    [key in keyof typeof productTypes]: Wallet[]
+};
+
+const VERY_HIGH_DATE_VALUE = 99999999999999;
+
+function resultAsPercent(usage: {balance: number; initialBalance: number}): number {
```

Comment:
It would probably make sense to place this closer to the related logic (e.g. ` totalUsageFromMultipleWallets`)